apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java

```diff
@@ -117,6 +118,7 @@ public void testDeleteSinglePartObjectWithMultiPartDelete() throws Exception {
          assertThat(resp.deleted()).isZero();
          assertThat(resp.notFound()).isZero();
          assertThat(resp.errors()).hasSize(1);
+         api.getObjectApi(regionId, containerName).delete(objectName);
```

Comment:
> Yeah, otherwise cleanup takes care of it.

Ah, OK. In that case, I'm probably confused: if cleanup takes care of it, why do we explicitly need to delete it here? Is that more efficient? Or do downstream tests fail otherwise? Or..?